qiime_studio/api/v1.py

```diff
@@ -115,3 +124,85 @@ def api_input_artifacts(plugin_name, workflow_name, input_name):
         if Artifact.load(artifact['path']).type <= input_type:
             input_artifacts.append(artifact)
     return jsonify({'input_artifacts': input_artifacts})
+
+
+@v1.route('/job/<plugin_name>/<workflow_name>', methods=['POST'])
+def execute_workflow(plugin_name, workflow_name):
+    plugin = PLUGIN_MANAGER.plugins[plugin_name]
+    workflow = plugin.workflows[workflow_name]
+
+    request_body = request.get_json()['jobData']
+
+    process_values = {}
+    for key, value in request_body.items():
+        if '-' in key:
+            type_, name = key.split('-')
+            process_values[name] = value
```

Comment:
`name` may clash between `inputs`, `parameters`, and `outputs`. I would recommend either splitting this into groups by `type_` or operating on the `signature` first and picking out the relevant pieces (constructing the key the same way as the UI did).